pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -115,18 +137,26 @@ func (m *Multicluster) AddMemberCluster(clientset kubernetes.Interface, metadata
 		ResyncPeriod: m.ResyncPeriod,
 		DomainSuffix: m.DomainSuffix,
 	}
+	webhookConfigName := strings.ReplaceAll(validationWebhookConfigNameTemplate, validationWebhookConfigNameTemplateVar, m.secretNamespace)
 	if m.fetchCaRoot != nil {
 		nc := NewNamespaceController(m.fetchCaRoot, opts, clientset)
 		go nc.Run(stopCh)
+		go util.PatchCertLoop(injectionWebhookConfigName, webhookName, m.caBundlePath, clientset, stopCh)
```

Comment:
should we guard this with centralIstiod before patch?

Reply:
m.fetchCaRoot will not be nil only if we are on centralIstiod, so this has been garded.